### PR TITLE
change to v2

### LINKED ISSUE #12: Add support to show knockout fixtures of Champions League as rounds
Explore, if possible, and add support to show knockout fixtures of Champions League as rounds

### LINKED ISSUE #45: Migrate project to V2 API
## Description
`football-data.org` has released version 2 of its API and is recommended to migrate to it.

Read more here:
https://www.football-data.org/blog

### PR DESCRIPTION
**Pull request by nilshah98**
## Description of new feature, or changes

- Currently, all the features from v1 migrated to v2.
- There are a few restrictions in the migrated version v2, while searching for fixtures and scores, it's only limited till 10 days.  
- LeagueID needs to be specified to extend filter for more than 10 days.

## ToDo

- [ ] Migrate fictures and scores, to matches URI
- [ ] Make league codes mandatory for scores and fixtures, for extending search day limit.

## Related Issues and Discussions

Fixes #45 

## People to notify

@manrajgrover

### CODE REVIEW

**Comment by nilshah98** [comment]
Here are the screen captures for the same -  
- `football standings -l PL`
y0rfxdaaq0
- `football fixtures -l PL -d 5`
y5qivemwnv
- No live matches right now, will update recording with scores soon.

File: config.json [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {
-  "API_KEY": ""
-}
+  "API_KEY": "8e6f6608e3154b1a956193827cdaec4c"
```
Comment (manrajgrover) [inline config.json]: You may want to change this now that it is in the public domain. Also, please be careful and not push keys to Github.

Comment (Freelancer-98) [reply config.json]:
```suggestion
  "API_KEY": "8e6f6608e3154b1a956193827cdaec4c"
```
Thanks, removed it.

Also, I am getting a lot of linting errors, any idea how to resolve those?
Currently using VS Code, on Windows 10.

Comment (nilshah98) [reply config.json]: Thanks, removed it.

Also, I am getting a lot of linting errors, any idea how to resolve those?
Currently using VS Code, on Windows 10.

**Comment by manrajgrover** [comment]
@Freelancer-98 It should be easy to configure VSCode. We already have `.vscode` settings defined. Please install `prettier` and `eslint` VSCode plugin.

**Comment by manrajgrover** [comment]
@Freelancer-98 Also, please elaborate on features migrated. Thanks for working on this one! 😄

**Comment by nilshah98** [comment]
> @Freelancer-98 Also, please elaborate on features migrated. Thanks for working on this one! 😄

All of the features mentioned in the readme have been migrated, except for the limitations on the number of days for fixtures and scores, which is 10 days, at the moment.  
Will try to work on it, as and when time permits.  

Was a pleasure, to work with the football-cli.

**Comment by nilshah98** [comment]
> @Freelancer-98 It should be easy to configure VSCode. We already have `.vscode` settings defined. Please install `prettier` and `eslint` VSCode plugin.

Cool, will look into this soon.

**Pull request closed without merging**